FluentAvalonia/Styling/Core/FluentAvaloniaTheme.axaml

```diff
@@ -0,0 +1,16 @@
+﻿<Styles
+    xmlns="https://github.com/avaloniaui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    x:Class="FluentAvalonia.Styling.FluentAvaloniaTheme">
+    <Styles.Resources>
+        <ResourceDictionary>
+            <FontFamily x:Key="SymbolThemeFontFamily">/Fonts/#Symbols</FontFamily>
+            <Color x:Key="SystemAccentColor">#00000000</Color>
+            <ResourceDictionary.MergedDictionaries>
+                <MergeResourceInclude Source="/Styling/StylesV2/Fluentv2.axaml" />
```

Comment:
Weird designer error
![image](https://user-images.githubusercontent.com/40413319/216860535-862963ce-4ff6-435e-a342-7abb8a70d7d6.png)

Also if you're going to switch to MergeResourceInclude, you need to go through all of FATheme and make sure any place that was expecting a merged dictionary is corrected. Though IMO we're not gaining that much with MergeResourceInclude here since it's just 1 xaml file that has to load and Fluentv2.axaml is basically already a merged xaml file, unlike upstream where their fluent theme is broken into like 5 different ResourceDictionaries.

Reply:
Done.